src/Slices/Notification/UI/Drawer/Drawer.tsx

```diff
@@ -111,35 +139,39 @@ const View: React.FC<ViewProps> = ({ data, onClose, trigger, drawerRef }) => {
       </NotificationDrawerHeader>
       <NotificationDrawerBody>
         <NotificationDrawerList>
-          {RemoteData.fold(
-            {
-              notAsked: () => null,
-              loading: () => null,
-              failed: () => null,
-              success: ({ data }) =>
-                data.map((notification) => (
-                  <Item
-                    data-testid="menuitem"
-                    {...{ notification }}
-                    key={notification.id}
-                    onUpdate={getOnUpdate([notification.id])}
-                  />
-                )),
-            },
-            data,
-          )}
+          {response.isSuccess
+            ? response.data.data.map((notification) => (
```

Comment:
i know there wasn't any typing before, but I think it would make sense to enforce the Notification typing in the different maps you've used in the file.

Reply:
but there is typing, if you hover over notification in the IDE.